NSS-Day-Cohort-42/rare-server-media-moguls
# Get all and Create categories methods

#### Changes Made
1. Added Categories class to the models folder
2. Added get_all_categories method to categories.py
3. Added create_category method to categories.py
4. updated request_handler.py do_GET method to include categories resource
5. updated request_handler.py do_POST method to include categories resource
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout sm-create-category-backend
```

2, Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
>Open Postman and
> Select GET from dropdown and make a request on http://localhost:8000/categories
> Check for status 200 is ok
>Select POST from dropdown and make a request on http://localhost:8000/categories
>Check for status 201 Created

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
